README.rst

```diff
@@ -268,6 +268,54 @@ There is another entry point for Android Studio - ``src/examples/facefilter/andr
 It should be used only for testing or as a template for starting your own project
 based on Drishti.
 
+Android Studio workarounds
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next factors add an instability to the Android Studio build:
+
+* Android Studio 3.2 is **beta**
+* Using custom CMake 3.7+ in Android Studio is `a preview feature <https://developer.android.com/studio/projects/add-native-code#vanilla_cmake>`__
+* Some issues are hard to track or confirm, some issues `like this one <https://issuetracker.google.com/issues/75268076>`__ is already reported but still **not fixed**
+
+From experience the weakest part in build is a communication between Gradle and
```

Comment:
From experience >>,<< the weakest part in >>the<< build is >>the<< communication between Gradle and CMake>>.<<  To minimize it, the >>following<< trick can be used: